src/coreclr/tools/superpmi/superpmi/streamingsuperpmi.cpp

```diff
@@ -0,0 +1,244 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "standardpch.h"
+#include "superpmi.h"
+#include "jitinstance.h"
+#include "simpletimer.h"
+#include "mclist.h"
+#include "lightweightmap.h"
+#include "commandline.h"
+#include "errorhandling.h"
+#include "methodcontext.h"
+#include "methodcontextreader.h"
+#include "spmiutil.h"
+#include "fileio.h"
+
+static bool ParseJitOption(const char* optionString, WCHAR** key, WCHAR** value)
```

Comment:
Can't we re-use the existing `ParseJitOption`?

Reply:
Probably, just need to do some refactoring.